hub/integrations/pytorch/dataset.py

```diff
@@ -508,33 +500,15 @@ def __iter__(self):
             self.torch_datset,
             batch_size=self.batch_size,
             num_workers=self.num_workers,
-            collate_fn=collate_fn,
+            collate_fn=identity,
```

Comment:
use identity function in inner loader and do actual collation in the outer loader